terraform/grafana-dashboard/dashboard.tf

```diff
@@ -0,0 +1,12 @@
+resource "grafana_dashboard" "aegis" {
+  folder = var.aegis_folder.uid
+  config_json = jsonencode({
+    title = "Aegis - On-chain Metrics"
+    time  = { from = "now-60m", to = "now" }
+    panels = concat(
+      local.trading_mode_panels,
+      local.oracle_relayer_panels,
+      local.legacy_client_panels,
```

Comment:
This dashboard is missing the aegis service panels that are are the bottom of the old dashboard: https://clabsmento.grafana.net/d/baec6799-0382-4c23-ae44-355300381bbe/aegis-on-chain-data

was that intentional?

Reply:
yeah, because 2 out of 3 panels are broken for me there, wasn't sure if anyone's still using those: 
<img width="1560" alt="image" src="https://github.com/user-attachments/assets/a7f126ea-cd94-40c5-81ae-2a5e7abd9524">

any idea why they're broken?